### ISSUE TITLE
AI doesn't work

### ISSUE THREAD

**Issue by KoujiHeneda** [tim-kos/the_duke_ai#1]
I tried to run the ai and there is an error which said as below:
the_duke_lib\buildings.per
ERROR 2005: Invalid Identifier: up-modify-sn

**Comment by tim-kos** [comment]
Hey there,Are you on AoE2: definitive edition?Kind regards,TimCo-Founder ***@***.***_kosAm 28.04.2024 um 20:19 schrieb KoujiHeneda ***@***.***>:﻿
I tried to run the ai and there is an error which said as below:
the_duke_lib\buildings.per
ERROR 2005: Invalid Identifier: up-modify-sn

**Comment by KoujiHeneda** [comment]
I'm not on the definite edition, I tried to run on the older edition which is the Conqueror 1.0c edition. But another error is saying that the list is full.

**Comment by tim-kos** [comment]
Yes this AI uses more than 40 goals.I am sorry, it is for the definitive edition, not conquerors. Also uses some functionality only available in DE.Kind regards,TimCo-Founder ***@***.***_kosAm 01.05.2024 um 04:46 schrieb KoujiHeneda ***@***.***>:﻿
I'm not on the definite edition, I tried to run on the older edition which is the Conqueror 1.0c edition. But another error is saying that the list is full.
